expenses-manager-backend/src/main/resources/data.sql

```diff
@@ -1,20 +1,42 @@
-INSERT INTO category (description, color) VALUES ('Food & Restaurants', '#ff6347');
-INSERT INTO category (description, color) VALUES ('Car', '#1e90ff');
```

Comment:
Fisierul asta poate are o versiune mai noua in main sau ceva, nu vad de ce sa adaugi tu la sql scripts, decat daca se asteapta sa faci tu asta aici

Reply:
Versiunea de aici am facut-o eu si am adaugat tichetul (cu partea asta) pe board-ul nostru. 😅
Cu versiunea din main nu ruleaza backend-ul pentru ca incearca sa adauge valori deja existente - problema aparuta martea trecuta, in call pe Discord.